media/js/base/sentry-init.js

```diff
@@ -16,12 +16,11 @@
             window.Sentry.init({
                 dsn: sentryDsn ,
                 sampleRate: 0.10,
-                denyUrls: [
-                    // Chrome extensions
-                    /extensions\//i,
-                    /^chrome:\/\//i,
-                    // Firefox extensions
-                    /^resource:\/\//i
+                ignoreErrors: [
+                    'https://plugin.ucads.ucweb.com/api/flow/'
+                ],
+                allowUrls: [
+                    '/media/js/'
```

Comment:
@pmac I think by only allowing errors from JS on specific URLs it should give us the best option for filtering out unwanted noise (e.g. from extensions and external scripts). The `allowedUrls` array here accepts multiple values and regex. By default it checks that a URL contains the value, so this should match for `https://www.mozilla.org/media/js/`, but also `https://foo.com/media/js/`. We could make it more strict, but maybe this is ok?

Thoughts?

Reply:
Oh interesting. So all errors are reported against the script URL and not the page one? In that case this looks great!